packages/ui/components/alert/alert.test.tsx

```diff
@@ -0,0 +1,57 @@
+import { render, screen } from "@testing-library/react";
+
+import { User as UserIcon } from "@calcom/ui/components/icon";
+
+import { Alert } from "./Alert";
+
+describe("Tests for Alert component", () => {
+  test("Should render text", () => {
+    render(<Alert severity="info" title="I'm an Alert!" message="Hello World" />);
+    expect(screen.getByText("I'm an Alert!")).toBeInTheDocument();
+    expect(screen.getByText("Hello World")).toBeInTheDocument();
+  });
+
+  test("Should have custom class name", () => {
+    render(<Alert severity="info" className="component-class" />);
+    expect(screen.getByTestId("alert").classList.toString()).toContain("component-class");
+  });
+
+  test("Should render custom icon with class names", () => {
+    render(
+      <Alert severity="info" CustomIcon={UserIcon} customIconColor="icon-color" iconClassName="icon-class" />
+    );
+
+    expect(screen.getByTestId("custom-icon")?.classList.toString()).toContain("icon-color");
+    expect(screen.getByTestId("custom-icon")?.classList.toString()).toContain("icon-class");
```

Comment:
Asserting classes set feels like it will lead to brittle tests and doesn't give a ton of value. I would suggest we don't add tests like these.